.github/workflows/audits.yml

```diff
@@ -96,7 +95,7 @@ jobs:
             *.cache-from=type=gha
             *.cache-to=type=gha,mode=max
       - name: Start
-        run: yarn workspace ${{ matrix.workspace }} start --detach --wait
+        run: yarn workspace ${{ matrix.workspace }} start --wait
```

Comment:
`--wait                      Wait for services to be running|healthy. Implies detached mode.`

Reply:
I know, it was intentionally there because I am not a fan if implications. :D